src/app/features/shared/components/organisms/DetailHeader/DetailHeader.tsx

```diff
@@ -0,0 +1,29 @@
+import React from "react"
+import styled from "styled-components"
+import { breakpoint } from "@datapunt/asc-ui"
+import Row from "app/features/shared/components/atoms/Grid/Row"
+import Column from "app/features/shared/components/atoms/Grid/Column"
+import BreadCrumbs from "app/features/addresses/components/molecules/BreadCrumbs/BreadCrumbs"
+import AddressDisplay from "app/features/addresses/components/atoms/AddressDisplay/AddressDisplay"
+type Props = {
+  bagId: string
+}
+
+const ColumnAlignRight = styled(Column)`
+@media screen and ${ breakpoint("min-width", "laptop") } {
+  text-align: right;
+}`
+
+
+const DetailHeader: React.FC<Props> = ({ bagId }) => (    
+  <Row>
+    <Column spanSmall={100} spanLarge={50}>
```

Comment:
Could we make the `spanSmall={100}` the default property value?

Reply:
Actually it is the default, so I removed it here